docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -38,27 +39,31 @@ A replication endpoint must exist before you create a replication rule. To creat
    `library/**`     | `library/hello-world`(Y)<br> `library/my/hello-world`(Y)
    `{library,goharbor}/**` | `library/hello-world`(Y)<br> `goharbor/harbor-core`(Y)<br> `google/hello-world`(N)
    `1.?`      | `1.0`(Y)<br> `1.01`(N)
-1. Use the **Destination Registry** drop-down menu to select from the configured replication endpoints. 
-1. Enter the name of the namespace in which to replicate resources in the **Destination namespace** text box.
+1. If you are creating a Push-based replication rule, use the **Destination Registry** drop-down menu to select from the configured replication endpoints.
+1. For **Destination Namespace**, enter the name of the namespace in which to replicate resources in the  text box. If you do not enter a namespace, resources are placed in the same namespace as in the source registry.
 
-   If you do not enter a namespace, resources are placed in the same namespace as in the source registry. 
-
-   ![Destination and namespaces](../../../img/replication-rule4.png)
-   
    **NOTE:** Because of major API changes in the v2.0 release to support [OCI](https://github.com/opencontainers/distribution-spec).
```

Comment:
IMO, moving the note in line 45 and 46  the beginning or the end of the doc is better, because it's a note for the whole replication functionality, not only for the `Destination Namespace` part

Reply:
i agree, i moved the note to the top of the page.